src/Polly/PublicAPI.Shipped.txt

```diff
@@ -0,0 +1 @@
+﻿
```

Comment:
For Polly, shouldn't we just put everything in `Shipped` now as we're (except type forwards) not changing anything here in v8?

Reply:
True, we might do that now. (except few recent additions)